packages/amplication-design-system/package.json

```diff
@@ -55,17 +57,20 @@
     "eslint-plugin-react-hooks": "4.6.0",
     "formik": "2.2.9",
     "prettier": "2.1.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
Why do this packages moved from peer

Reply:
We don't have cyclic dependencies here, we don't need peer dependencies.